templates/work_item.md

```diff
@@ -7,9 +7,15 @@ participants.
 - Full Name (@github, email@example.com, or both)
 - ...
 
+## External Partners (OPTIONAL)
+**This is only for work items intended to create a standards proposal. This may
+be filled in later and is not required for initial approval of the work item.**:
```

Comment:
I think I get what you are saying, perhaps this wording would be more clear:
```suggestion
**This is only for work items intended for transition to an external standards
body after incubation here. This may be filled in later and is not required for
initial approval of the work item.**:
```